mypy/semanal.py

```diff
@@ -3680,7 +3683,14 @@ def analyze(self, type: Type) -> None:
             analyzer = TypeAnalyserPass3(self.fail)
             type.accept(analyzer)
 
-    def fail(self, msg: str, ctx: Context, *, blocker: bool = False) -> None:
+    def fail(self, msg: str, ctx: Context, *, blocker: bool = False,
+             implicit_any: bool = False) -> None:
+        if implicit_any:
+            if not self.options.warn_implicit_any or self.errors.file.endswith('.pyi'):
```

Comment:
I also can't find it. 

In stub files, a lot of the times the type argument in genertic types is omitted (e.g., `Union[type, Tuple]`, `def f() -> tuple:`, `def g() -> Iterable:`, etc.). There'll be hundreds of warnings like this, making the flag unusable. 
I am sure we don't want to change the stubs to avoid implicit `Any` (and even if we did, that change would have to happen first, before `warn_implicit_any` starts paying attention to stubs).